command/output.go

```diff
@@ -7,34 +7,34 @@ import (
 // OutputFormatter is the standardized interface all output formatters
 // should use
 type OutputFormatter interface {
-	// getErrorOutput returns the CLI command error
-	getErrorOutput() string
-
-	// getCommandOutput returns the CLI command output
-	getCommandOutput() string
-
```

Comment:
It does not make sense to have these funcs as part of the interface because they are used within the package only and as helper functions